assets/promptflow/models/foundation-bring-your-own-data-chat-qna/model.yaml

```diff
@@ -0,0 +1,8 @@
+path:
+  container_name: models
+  container_path: samples/foundation_bring_your_own_data_chat_qna/v1/model_folder
```

Comment:
@gjwoods have you done the CELA review for these two new added flow samples ? If not, please refer [this doc](https://msdata.visualstudio.com/Vienna/_git/PromptFlow?path=/docs/sharing-your-flows-in-prompt-flow-gallery.md&_a=preview) to get your flows prepared. And please notice that, do not merge this PR before getting the CELA approval for these two flows. Thank you !